Core/src/org/sleuthkit/autopsy/modules/interestingitems/FilesSet.java

```diff
@@ -737,6 +755,46 @@ public boolean passes(AbstractFile file) {
 
         }
 
+        /**
+         * A class for checking whether a file's creation or modification
+         * occured in a specific range of time
+         */
+        static final class DateCondition implements FileAttributeCondition {
+
+            private final static long DAY_IN_S = 60 * 60 * 24;
+
+            private int daysIncluded;
+
+            /**
+             * Construct a new DateCondition
+             *
+             * @param days - files created or modified more recently than this
+             *             number of days will pass
+             */
+            DateCondition(int days) {
+                daysIncluded = days;
+            }
+
```

Comment:
What are the valid values for daysIncluded?  The UI allows for 0 as well as a negative number.

Reply:
Modifying it to allow 0 (just in case they want to see if any file has a bogus future modification or creation date) but to not allow negative numbers because they just add a layer of confusion, I can easily disallow 0 as well if that is deemed simpler for the user.